packages/shared/src/components/squads/SquadPostsProgressBar.tsx

```diff
@@ -42,7 +42,7 @@ export const SquadPostsProgressBar = ({
           Posts
         </span>
         <span className={classNames(className?.labelContent ?? 'font-bold')}>
-          {effectivePostsCount} / {goal}
```

Comment:
From Figma, there should be no spaces in between. However, what do you think looks better?

<img width="289" alt="Screenshot 2024-05-19 at 9 09 19 PM" src="https://github.com/dailydotdev/apps/assets/13744167/f27c399b-fae2-4409-bce3-a00a21776b0a">
<img width="240" alt="Screenshot 2024-05-19 at 8 05 45 PM" src="https://github.com/dailydotdev/apps/assets/13744167/5f8fd5a7-6c23-4e6b-a59c-3c25f19e46a5">

Reply:
I think I'm fine with the no space